### PR TITLE
Properly load vcards from disk

### PR DESCRIPTION
**Pull request by ryscheng**
Addresses
https://github.com/uProxy/uproxy/issues/1267

### CODE REVIEW

File: src/core/google-auth.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*globals freedom:true,setTimeout,VCardStore,XMPPSocialProvider */
+/*globals freedom:true,setTimeout,VCardStore,XMPPSocialProvider,console */
```
Comment (willscott) [inline src/core/google-auth.js]: console needed?

File: src/core/vcardstore.js [hunk 2]
```diff
@@ -167,48 +147,72 @@ VCardStore.prototype.updateProperty = function(user, property, value) {
...
+      console.warn(e);
+    }
+  };
+
+  if (!this._storage) {
+    return;
+  }
+
+  this._storage.keys().then(function(keys) {
+    for(var i=0; i<keys.length; i++) {
```
Comment (willscott) [inline src/core/vcardstore.js]: spacing

File: src/core/vcardstore.js [hunk 3]
```diff
@@ -167,48 +147,72 @@ VCardStore.prototype.updateProperty = function(user, property, value) {
...
     this.loadCard(next);
   } else {
-    setTimeout(this.checkVCardQueue.bind(this), this.THROTTLE_TIMEOUT);
+    setTimeout(this._checkVCardQueue.bind(this), this.THROTTLE_TIMEOUT);
   }
 };
+
+VCardStore.prototype._init = function() {
+  var userId, profile;
+  var tryLoad = function(k, v) {
```
Comment (willscott) [inline src/core/vcardstore.js]: this function may need to be bound to get appropriate this?

**Comment by willscott** [comment]
:+1:

File: src/core/google-auth.js (hunk 1 above)
Comment (ryscheng) [inline src/core/google-auth.js]: Done

File: src/core/vcardstore.js (hunk 2 above)
Comment (ryscheng) [inline src/core/vcardstore.js]: Done

File: src/core/vcardstore.js (hunk 3 above)
Comment (ryscheng) [inline src/core/vcardstore.js]: Agreed this is fine out of band

**Pull request merged**